src/js/settings.js

```diff
@@ -0,0 +1,77 @@
+export function saveLanguageChoice() {
+	// does not change the language atm but the value is saved
```

Comment:
E in regula, o sa facem un task separat in care facem sa se schimbe si limba. Marcheaza aici in cod cu to do.